distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -196,7 +206,7 @@ Text.is_match : Text -> (Text_Matcher | Regex_Matcher) -> Boolean ! Regex.Compil
 Text.is_match self pattern=".*" matcher=Regex_Matcher.Regex_Matcher_Data = case matcher of
     Text_Matcher.Case_Sensitive -> self == pattern
     Text_Matcher.Case_Insensitive locale -> self.equals_ignore_case pattern locale
-    _ : Regex_Matcher.Regex_Matcher ->
+    _ : Regex_Matcher ->
```

Comment:
I really like how this piece of code evolved when type-matching and now this refactor were added. Looks much better than in the past!